google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -47,10 +47,52 @@ class Bucket
         attr_accessor :gapi
 
         ##
+        # A `true`/`false` value or a project ID string to be sent as the
```

Comment:
FWIW, I understand it's described in the next paragraph, but they are introduced here.